res/controllers/Traktor-Kontrol-S4-MK3.js

```diff
@@ -0,0 +1,193 @@
+/* eslint no-redeclare: "off", no-unused-vars: "off" */
```

Comment:
I would prefer if you did this locally instead of disabling for the entire file, at least when the mapping is finished, I guess you can keep it if its gets too annoying during development.
https://eslint.org/docs/user-guide/configuring/rules#using-configuration-comments-1